coalib/processes/Processing.py

```diff
@@ -303,7 +309,15 @@ def filter_raising_callables(it, exception, *args, debug=False, **kwargs):
     """
     for elem in it:
         try:
+            if debug_bears is False:
+                kwargs['debug_flag'] = False
+            elif ((str(debug_bears[0]).lower().strip())=='true') or \
+                ((elem.__name__) in debug_bears):
+                kwargs['debug_flag'] = True
+            else:
+                kwargs['debug_flag'] = False
             yield elem(*args, **kwargs)
+            kwargs.pop('debug_flag')
```

Comment:
just leave it there ;)

Reply:
It's working because it's overwriting the `debug_flag` `kwarg`.

Reply:
I think I was making some mistake previously then :+1: